lib/beaker-hostgenerator/data.rb

```diff
@@ -1,11 +1,21 @@
 module BeakerHostGenerator
   module Data
 
-    # Pull various informations out of the environment.
-    PE_VERSION=ENV['pe_version']
-    PE_FAMILY=ENV['pe_family']
-    PE_UPGRADE_VERSION=ENV['pe_upgrade_version']
-    PE_UPGRADE_FAMILY=ENV['pe_upgrade_family']
+    def pe_version
+      ENV['pe_version']
```

Comment:
Do we want to memoize calls to the environment here or do we expect these values to change?